eng/Signing.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <!-- Signing.props is a required file for the signing validation step. -->
   <ItemGroup>
-
+    <FileSignInfo Include="MessagePack.Annotations.dll" CertificateName="3PartySHA2" />
+    <FileSignInfo Include="MessagePack.dll" CertificateName="3PartySHA2" />
```

Comment:
The compilers added new dependencies?

Reply:
Microsoft.CodeAnalysis.Features add a dependency on Microsoft.VisualStudio.Debugger.Contracts which I assume added these dependencies.